src/main/java/ua/team3/carsharingservice/exception/InvalidPaymentTypeException.java

```diff
@@ -0,0 +1,7 @@
+package ua.team3.carsharingservice.exception;
+
+public class InvalidPaymentTypeException extends RuntimeException {
```

Comment:
I'm not sure if you added it to GlobalExceptionHandler

Reply:
might have been erased due to conflicts, I'll check later